specification/azsadmin/resource-manager/containerregistry/Microsoft.ContainerRegistry.Admin/preview/2019-11-01/quotas.json

```diff
@@ -0,0 +1,271 @@
+          }
+        },
+        "tags": [
+          "ContainerRegistryQuotas"
+        ],
+        "description": "Returns the specified container registry quota.",
+        "operationId": "ContainerRegistryQuotas_Get",
+        "parameters": [
+          {
+            "$ref": "containerregistry.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "containerregistry.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "containerregistry.json#/parameters/LocationParameter"
+          },
+          {
+            "$ref": "#/parameters/QuotaNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- The container registry quota has been returned.",
+            "schema": {
+              "$ref": "#/definitions/ContainerRegistryQuota"
+            }
+          },
+          "default": {
+            "description": "Error Response.",
+            "schema": {
+              "$ref": "containerregistry.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.ContainerRegistry.Admin/locations/{location}/quotas": {
+      "get": {
+        "x-ms-examples": {
+          "Returns a list of container registry quotas at the given location.": {
+            "$ref": "./examples/Quotas/List.json"
+          }
+        },
+        "tags": [
+          "ContainerRegistryQuotas"
+        ],
+        "description": "Returns a list of container registry quotas at the given location.",
+        "operationId": "ContainerRegistryQuotas_List",
+        "parameters": [
+          {
+            "$ref": "containerregistry.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "containerregistry.json#/parameters/LocationParameter"
+          },
+          {
+            "$ref": "containerregistry.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- The list of container registry quotas has been returned.",
+            "schema": {
+              "$ref": "#/definitions/ContainerRegistryQuotaList"
+            }
+          },
+          "default": {
+            "description": "Error Response.",
+            "schema": {
+              "$ref": "containerregistry.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ContainerRegistryQuota": {
+      "description": "Container registry quota.",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "description": "Container registry quota properties.",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ContainerRegistryQuotaProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "containerregistry.json#/definitions/Resource"
```

Comment:
use the appropriate common resource type instead.

Reply:
we use "containerregistry.json for shared definitions and Resource is part of them

Reply:
@markcowl Is there a better common Resource? We have been defining the Resource ourselves in each of the admin specs

Reply:
Yes, you should use the common resource definitions here https://github.com/Azure/azure-rest-api-specs/blob/master/specification/common-types/resource-management/v2/types.json#L9-L84  - you can have your resource types reference these schemarather than having your own resource.  This promotes uniformaity across apis.